021-team-credential-managers/proposal.md

```diff
@@ -0,0 +1,128 @@
+# Summary
+
+Move credential management configuration to the team configuration level, and
+allow configuring multiple credential managers for each team.
```

Comment:
Chatting with @pivotal-jwinters led to potentially thinking of this as something that's per-*pipeline*, not just per-*team*. I haven't put any technical thinking into it yet, but I do really like it from the viewpoint that it allows for even safer scoping of credential access (i.e. we don't have to worry about our "prs" pipeline having access to prod credentials), and also makes pipelines more self-contained, which is the single most important thing about pipelines.

Reply:
This should also be scoped to projects if you implement it.